### PR TITLE
Disable really large Data tests that take a bit too long to run

### PR DESCRIPTION
**Pull request by phausler**
The benchmarks for Data are a bit harsh for CI to run. This disables the really big ones.

### CODE REVIEW

**Comment by phausler** [comment]
@swift-ci please smoke test

File: benchmark/single-source/DataBenchmarks.swift [hunk 1]
```diff
@@ -17,49 +17,49 @@ public let DataBenchmarks = [
...
     BenchmarkInfo(name: "DataCreateEmptyArray", runFunction: run_createEmptyArray, tags: [.validation, .api, .Data]),
     BenchmarkInfo(name: "DataCreateSmallArray", runFunction: run_createSmallArray, tags: [.validation, .api, .Data]),
     BenchmarkInfo(name: "DataCreateMediumArray", runFunction: run_createMediumArray, tags: [.validation, .api, .Data]),
     BenchmarkInfo(name: "DataSubscriptSmall", runFunction: run_SubscriptSmall, tags: [.validation, .api, .Data]),
     BenchmarkInfo(name: "DataSubscriptMedium", runFunction: run_SubscriptMedium, tags: [.validation, .api, .Data]),
-    BenchmarkInfo(name: "DataSubscriptLarge", runFunction: run_SubscriptLarge, tags: [.validation, .api, .Data]),
+    BenchmarkInfo(name: "DataSubscriptLarge", runFunction: run_SubscriptLarge, tags: [.validation, .api, .Data, .skip]),
     BenchmarkInfo(name: "DataCountSmall", runFunction: run_CountSmall, tags: [.validation, .api, .Data]),
     BenchmarkInfo(name: "DataCountMedium", runFunction: run_CountMedium, tags: [.validation, .api, .Data]),
     BenchmarkInfo(name: "DataCountLarge", runFunction: run_CountLarge, tags: [.validation, .api, .Data]),
```
Comment (eeckstein) [inline benchmark/single-source/DataBenchmarks.swift]: Can you also disable this one. It also takes long.

**Review by eeckstein — CHANGES_REQUESTED** [review]
Please also disable DataCountLarge

**Comment by phausler** [comment]
@swift-ci please smoke test and merge

**Comment by airspeedswift** [comment]
While we're looking, `IterateData` is very noisy – it appears in about half of my benchmark runs.

**Comment by phausler** [comment]
@swift-ci please smoke test

**Review by eeckstein — APPROVED** [review]

**Comment by gottesmm** [comment]
@phausler just a suggestion. Maybe we should have a .long tag or something like that (and maybe run them during the long test?) not sure.

**Comment by palimondo** [comment]
@airspeedswift Re: noisy `IterateData`. With what benchmark running method are you seeing this?

**Pull request merged**